website/project/views/comment.py

```diff
@@ -49,6 +49,7 @@ def update_file_guid_referent(self, target, event_type, payload, user=None):
             old_file = BaseFileNode.load(obj.referent._id)
             obj.referent = create_new_file(obj, source, destination, destination_node)
             obj.save()
+            obj.referent.reload()
```

Comment:
hmm, why do we need to reload the referent object when we are not using it at all afterwards?